applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_compressible_perturbation_potential_element.cpp

```diff
@@ -67,40 +146,63 @@ KRATOS_TEST_CASE_IN_SUITE(CompressiblePerturbationPotentialFlowElementRHS, Compr
     GenerateCompressiblePerturbationElement(model_part);
     Element::Pointer pElement = model_part.pGetElement(1);
 
-    AssignPotentialsToNormalCompressiblePerturbationElement(pElement);
+    std::array<double, 3> potential{1.0, 20.0, 50.0};
+    AssignPotentialsToNormalCompressiblePerturbationElement(pElement, potential);
 
     // Compute RHS
     Vector RHS = ZeroVector(3);
 
     const ProcessInfo& r_current_process_info = model_part.GetProcessInfo();
     pElement->CalculateRightHandSide(RHS, r_current_process_info);
 
-    std::vector<double> reference{146.2643261263345,-122.1426284341492,-24.12169769218525};
+    std::vector<double> reference{131.4361747323354,-113.768439084114,-17.66773564822145};
 
     KRATOS_CHECK_VECTOR_NEAR(RHS, reference, 1e-13);
 }
 
-/** Checks the CompressiblePerturbationPotentialFlowElement.
- * Checks the LHS computation.
- */
+KRATOS_TEST_CASE_IN_SUITE(CompressiblePerturbationPotentialFlowElementRHSClamping, CompressiblePotentialApplicationFastSuite) {
+    Model this_model;
+    ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+
+    GenerateCompressiblePerturbationElement(model_part);
+    Element::Pointer pElement = model_part.pGetElement(1);
```

Comment:
```suggestion
    Element::Pointer p_element = model_part.pGetElement(1);
```
Minor comment:
Just to say that this does not follow the style-guide (it is wrong in mostly all tests). Probably better to do a separate PR cleaning up everything

Reply:
Ok! Ill do it in an another PR as you say!